<launch>
  <node name="lidar_undistortion_node" pkg="lidar_undistortion"  type="lidar_undistortion" output="screen">

    <param name="lidar_topic" value="/scan"/>
    <!-- Estimated time of lidar message transmission(ms) -->
    <param name="lidar_msg_delay_time" value="0"/>

    <param name="scan_direction_clockwise" value="true"/>

    <param name="imu_topic" value="/imu"/>
    <param name="imu_frequency" value="50.0"/>

    <param name="output_frame_id" value="laser_frame"/>
    <!-- <param name="pub_raw_scan_pointcloud" value="true"/> -->
    <param name="scan_direction_clockwise_" value="true"/>



    <param name="pub_laserscan" value="true"/>
    <param name="laserscan_angle_increment" value="0.0069089"/>    <!--#0.008373 #0.0187-->

    <param name="use_range_filter" value="false"/>
    <param name="range_filter_min" value="0.1"/>
    <param name="range_filter_max" value="16.0"/>

    <param name="use_angle_filter" value="true"/>
    <param name="angle_filter_min" value="-3.1415926536"/>
    <param name="angle_filter_max" value="3.1415926536"/>

    <param name="use_radius_outlier_filter" value="false"/>
    <param name="radius_outlier_filter_search_radius" value="3"/>
    <param name="radius_outlier_filter_min_neighbors" value="5"/>

  </node>
  <!-- <node pkg="lidar_undistortion" type="line_fitting_node" name="line_fitting_node" output="screen"/> -->

  <!-- <node pkg="lidar_undistortion" type="line_fitting_new" name="line_fitting_new" output="screen"/> -->



</launch>
